src/deroff.rs

```diff
@@ -1182,27 +1201,31 @@ fn test_do_tbl() {
     assert!(deroffer.tblTab.is_empty());
     assert_eq!(deroffer.s, ";Hello World!");
     assert_eq!(deroffer.output.take(), "\n");
+    assert_eq!(deroffer.tblstate, TblState::Format);
```

Comment:
I think we should arrange all of these in an order to make it easier to read. Or even better, use a macro. But I think this is sufficient.

Reply:
I'm converting them to a macro right now :)

Reply:
macro version is uglier, we're gonna keep it how it is :smile: